index.template.html

```diff
@@ -2311,6 +2311,88 @@ <h3><code>securityDefinitions</code> and <code>security</code></h3>
 Using a specific prefix as in the above example
 for URI variables declared in security schemes can make it easier to avoid name conflicts.
 </p>
+<p>Security parameters might also be included along with the payload in some
+systems.  For example, suppose a system requires every payload to be a JSON
+object including a member named <code>key</code> whose value is a secret key.
```

Comment:
if make deeper make sure to update this too.   Also mention the correct syntax for JSON pointer somewhere.